src/main/java/uk/gov/hmcts/reform/prl/services/ManageOrderService.java

```diff
@@ -1344,6 +1351,39 @@ private void servedC100Order(CaseData caseData, List<Element<OrderDetails>> orde
         );
     }
 
+    private List<Element<ServedParties>> getServedParties(CaseData caseData) {
+        List<Element<ServedParties>> servedParties = new ArrayList<>();
+        if (caseData.getManageOrders()
+            .getRecipientsOptions() != null) {
+            servedParties = dynamicMultiSelectListService
+                .getServedPartyDetailsFromDynamicSelectList(caseData
+                                                                .getManageOrders()
+                                                                .getRecipientsOptions());
+        } else if (caseData.getManageOrders()
+            .getRecipientsOptionsOnlyC47a() != null) {
+            servedParties = dynamicMultiSelectListService
+                .getServedPartyDetailsFromDynamicSelectList(caseData.getManageOrders()
+                                                         .getRecipientsOptionsOnlyC47a());
+        }
+        if (C100_CASE_TYPE.equalsIgnoreCase(CaseUtils.getCaseTypeOfApplication(caseData))
+            && caseData.getManageOrders().getChildOption() != null) {
+            servedParties.addAll(dynamicMultiSelectListService
+                                     .getServedPartyDetailsFromDynamicSelectList(caseData.getManageOrders()
+                                                                                     .getChildOption()));
+        }
+
+        if (caseData.getManageOrders().getOtherParties() != null) {
```

Comment:
Other party applies only for C100 case, should we add check here?

Reply:
@shashisk97 Added condition to add other parties only in case of C100